com.reprezen.swagedit/src/com/reprezen/swagedit/schema/ObjectTypeDefinition.java

```diff
@@ -63,7 +63,22 @@ private void init() {
         initRequired();
         initProperties("properties", properties);
         initProperties("patternProperties", patternProperties);
-        initProperties("additionalProperties", additionalProperties);
+
+        if (content.has("additionalProperties") && content.get("additionalProperties").isObject()) {
+            JsonNode properties = content.get("additionalProperties");
```

Comment:
Is this because top-level parameters are defined as 
```json
 "parameterDefinitions": {
      "type": "object",
      "additionalProperties": {
        "$ref": "#/definitions/parameter"
      },
      "description": "One or more JSON representations for parameters"
    }
```
?